index.html

```diff
@@ -637,13 +637,15 @@ <h3>Proofs</h3>
           <dt><dfn class="lint-ignore">created</dfn></dt>
           <dd>
 The date and time the proof was created is OPTIONAL and, if included, MUST be
-specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string.
+specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string and SHOULD include a
+zone offset, preferably Z (meaning UTC).
```

Comment:
Hmm, I believe the change is redundant.  `dateTimeStamp` values MUST include a zone offset per https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp

We also shouldn't deviate from the guidance given in the vc-data-model, namely:

https://w3c.github.io/vc-data-model/#representing-time

You might look at the definition for validity periods in vc-data-model for appropriate wording: https://w3c.github.io/vc-data-model/#validity-period